code/game/objects/structures/spawner.dm

```diff
@@ -14,6 +14,44 @@
 	var/spawn_text = "emerges from"
 	var/faction = list(FACTION_HOSTILE)
 	var/spawner_type = /datum/component/spawner
+	/// Is this spawner taggable with a mining scanner?
+	var/scanner_taggable = FALSE
+	/// Has this spawner been tagged/analyzed by a mining scanner?
+	var/gps_tagged = FALSE
+	/// A short identifier for the mob it spawns. Keep around 3 characters or less?
+	var/mob_gps_id = "???"
+	/// A short identifier for what kind of spawner it is, for use in putting together its GPS tag.
+	var/spawner_gps_id = "Creature Nest"
+	/// A complete identifier. Generated on tag (if tagged), used for its examine.
+	var/assigned_tag
+
+/obj/structure/spawner/examine(mob/user)
+	. = ..()
+	if(scanner_taggable)
```

Comment:
```suggestion
	if(!scanner_taggable)
		return
```
If it's more than one or two lines you're intenting, try to invert the check and return out of the proc instead.